packages/reporters/dev-server/src/HMRServer.js

```diff
@@ -29,6 +34,8 @@ type HMRMessage =
       |},
     |};
 
+const FS_CONCURRENCY = 64;
```

Comment:
https://github.com/parcel-bundler/parcel/blob/069a2b1d773cb9fe02d621c2a281ecb47b5bf668/packages/shared/scope-hoisting/src/concat.js#L71

https://github.com/parcel-bundler/parcel/blob/069a2b1d773cb9fe02d621c2a281ecb47b5bf668/packages/packagers/js/src/JSPackager.js#L84

https://github.com/parcel-bundler/parcel/blob/069a2b1d773cb9fe02d621c2a281ecb47b5bf668/packages/packagers/css/src/CSSPackager.js#L21

Should we make this limit consistent? Is there a reason it's twice as large here?

Reply:
One difference here is this is run in a reporter on a single thread, whereas packagers can run concurrently across threads, each with their own queue of open files.